project/AppDependencies.scala

```diff
@@ -23,16 +23,16 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"   %% "bootstrap-frontend-play-27" % "3.4.0",
-    "uk.gov.hmrc"   %% "play-frontend-hmrc"         % "0.40.0-play-27",
+    "uk.gov.hmrc"   %% "bootstrap-frontend-play-27" % "4.0.0",
```

Comment:
they build bootstrap-frontend 2 days ago. I am not sure if it is ready for prod.